tests/test_payment_platforms.py

```diff
@@ -91,8 +93,10 @@ async def test_no_matching_account_raises(monkeypatch):
     )
 
     with pytest.raises(NoPaymentAccountError) as exc_info:
-        async with Golem(
-            budget=10.0, payment_driver="matching-driver", payment_network="matching-network"
+        async with GolemFactory(
+            budget=10.0,
```

Comment:
I guess, with a `GolemFactory` in place, we could actually add `budget` there so that it doesn't need to be specified explicitly anymore...

Reply:
done